CatAnalyzer/test/TTdilepDraw/topDraw.py

```diff
@@ -7,24 +7,27 @@
 topDraw.py -a 1 -s 1 -c 'tri==1&&filtered==1' -b [40,0,40] -p nvertex -x 'no. vertex' &
 topDraw.py -a 1 -s 1 -b [100,-3,3] -p lep1_eta,lep2_eta -x '#eta' &
 '''
-datalumi = 2.1114 #Run2015 v765
+datalumi = 37.06 #Run2016 v806
```

Comment:
https://hypernews.cern.ch/HyperNews/CMS/get/luminosity/688.html
With latest normtag: totalrecorded = 35.922 /fb, with uncertainty of 2.5%
(36.8 /fb is quite common, though...)